<p>Errore durante il ripristino dell'istanza <strong>{{ $instance_name }}</strong> dal backup. Messaggio: {{ $exception_message }}
</p>
